crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -59,6 +62,43 @@ pub async fn main(command: &DaemonCommand, base: &CommandBase) -> anyhow::Result
     Ok(())
 }
 
+pub async fn daemon_server(
+    base: &CommandBase,
+    idle_time: &Option<String>,
+) -> Result<(), DaemonError> {
+    let log_file = {
+        let directories = directories::ProjectDirs::from("com", "turborepo", "turborepo")
+            .expect("user has a home dir");
+
+        let folder =
+            turborepo_paths::AbsoluteNormalizedPath::new(directories.data_dir()).expect("absolute");
+
+        let hash = format!("{}-turbo.log", base.repo_hash());
+
+        let logs = turborepo_paths::ForwardRelativePath::new("logs").expect("forward relative");
+        let file = turborepo_paths::ForwardRelativePath::new(&hash).expect("forward relative");
+
+        folder.join(logs).join(file)
+    };
+
+    let repo_root =
+        turborepo_paths::AbsoluteNormalizedPathBuf::new(base.repo_root.clone()).expect("absolute");
```

Comment:
Not necessarily related to your change, but we should probably be storing the repo root as an `AbsoluteNormalizedPathBuf`

Reply:
Lets discuss today, want to get the semantics right here.